sechub-webui/src/main/java/com/mercedesbenz/sechub/webui/IndexController.java

```diff
@@ -6,14 +6,16 @@
 import org.springframework.ui.Model;
 import org.springframework.web.bind.annotation.GetMapping;
 
+import com.mercedesbenz.sechub.webui.configuration.SecHubAccessService;
+
 @Controller
 public class IndexController {
     @Autowired
-    SecHubServerAccessService accessService;
+    SecHubAccessService accessService;
 
     @GetMapping("/")
     String index(Model model) {
-        model.addAttribute("sechubServerUrl", accessService.getSecHubServerUrl());
+        model.addAttribute("sechubServerUrl", accessService.getSecHubServerUri());
```

Comment:
you changed the getter to "uri" - should the attribute sechubServerUrl" be renamed as well?